index.html

```diff
@@ -4,16 +4,91 @@
 <head>
   <meta charset="utf-8">
 
-  <title>My Resume</title>
+  <title>LV Wood Resume</title>
 
   <link rel="stylesheet" href="css/index.css">
+  <link href="https://fonts.googleapis.com/css?family=Montserrat|Playfair+Display" rel="stylesheet">
 
   <!--[if lt IE 9]>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
```

Comment:
Nice addition of Google Fonts! For compliance with web standards on URL encoding, use `%7C` instead of the `|` character in your font URL's (this becomes more noticable when using an HTML validator).